### ISSUE TITLE
Tapsigner does not support taproot (yet)

### ISSUE THREAD

**Issue by scgbckbone** [nunchuk-io/nunchuk-mobile-issues#9]
Just tested  Tapsigner (1.0.0) with nunchuck on android 11. Works great!!!

One thing that I think is bad from UX perspective is the ability to add tapsigner nfc key to taproot wallet. Tapsigner is not able to sign taproot inputs (no schnorr support yet). Showing just warning, to check compatibility of your HW devices is imho not enough. Users do stupid things all the time.

Coins that user sends to taproot wallet with nfc tapsigner key are effectively locked. When attempting to spend taproot utxo nunchuck flashes that `transaction signed successfully` but nothing is signed and button to broadcast transaction does not appear.

It would be better, if user wants to add tapsigner to taproot wallet, error should be thrown describing the limitations.

**Comment by hugohn** [comment]
Yup Coinkite does not support Taproot signing yet. Showing an error is a good suggestion. Thanksl @scgbckbone

**Comment by scgbckbone** [comment]
same applies to Coldcard
